acceptance/suites/tests/service_framework/reload_updated_crl.rb

```diff
@@ -25,8 +30,12 @@
         "then mv -f #{inventory_file}.bak #{inventory_file}; fi")
   end
 
+  step 'Disable crl reloading in bootstrap.cfg' do
+    on(master, "sed -i 's,#{watcher_service},##{watcher_service},' #{bootstrap}")
```

Comment:
Should we validate that the defaults are what we expect? Like, check that, in this branch, we have commented out the service before continuing? Potentially making that check prior to skipping if not on Java 8?